src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/attachment/v5_4/OpenNebulaAttachmentPluginTest.java

```diff
@@ -19,72 +19,61 @@
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
-import cloud.fogbow.common.exceptions.FogbowException;
-import cloud.fogbow.common.exceptions.InstanceNotFoundException;
-import cloud.fogbow.common.exceptions.InvalidParameterException;
-import cloud.fogbow.common.exceptions.UnexpectedException;
 import cloud.fogbow.common.models.CloudUser;
-import cloud.fogbow.common.models.SystemUser;
-import cloud.fogbow.common.models.linkedlists.SynchronizedDoublyLinkedList;
-import cloud.fogbow.common.util.HomeDir;
-import cloud.fogbow.ras.constants.SystemConstants;
 import cloud.fogbow.ras.core.SharedOrderHolders;
 import cloud.fogbow.ras.core.models.orders.AttachmentOrder;
 import cloud.fogbow.ras.core.models.orders.ComputeOrder;
-import cloud.fogbow.ras.core.models.orders.OrderState;
 import cloud.fogbow.ras.core.models.orders.VolumeOrder;
 import cloud.fogbow.ras.core.plugins.interoperability.opennebula.OpenNebulaClientUtil;
 import cloud.fogbow.ras.core.plugins.interoperability.opennebula.OpenNebulaStateMapper;
 
+import static cloud.fogbow.ras.core.plugins.interoperability.opennebula.attachment.v5_4.OpenNebulaAttachmentPlugin.IMAGE_ID_PATH_FORMAT;
+import static cloud.fogbow.ras.core.plugins.interoperability.opennebula.attachment.v5_4.OpenNebulaAttachmentPlugin.TARGET_PATH_FORMAT;
+
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({OpenNebulaClientUtil.class, SharedOrderHolders.class, VirtualMachine.class})
-public class OpenNebulaAttachmentPluginTest {
-
-	private static final String CLOUD_NAME = "opennebula";
-	private static final String DEFAULT_DEVICE_PREFIX = "vd";
-	private static final String FAKE_DEVICE = "fake-image-device";
-	private static final String FAKE_ID_PROVIDER = "fake-id-provider";
-	private static final String FAKE_INSTANCE_ID = "1";
-	private static final String FAKE_NAME = "fake-name";
-	private static final String FAKE_PROVIDER = "fake-provider";
-	private static final String FAKE_USER_ID = "fake-user-id";
+@PrepareForTest({DatabaseManager.class, OpenNebulaClientUtil.class, SharedOrderHolders.class, VirtualMachine.class})
```

Comment:
**SharedOrderHolders**.class can be removed from this line as there is no more mock static for this class.

Reply:
done.